schema/tracer_provider.json

```diff
@@ -30,5 +42,80 @@
                 }
             }
         }
+    },
+    "$defs": {
+        "SimpleSpanProcessor": {
```

Comment:
What do you think of omitting SimpleSpanProcessor? It's kind of a footgun that a lot of users get hung up on

Reply:
It's definitely an option, i included because its a nice to have when starting, but its not necessary

Reply:
Situations like a Lambda function.

Is it a footgun because of the name? Maybe should have been BlockingSpanProcessor or something :)

Reply:
Its a footgun because it triggers one export request per span. Extremely noisy!

Reply:
> Situations like a Lambda function.

They could also force flush their BatchSpanProcessor at the end of the function. At least that's what we've been recommending for GCP serverless. @tsloughter would this not work for Lambda?

> Is it a footgun because of the name? Maybe should have been BlockingSpanProcessor or something :)

The performance issues from blocking the calling thread have been a really problematic trap that new users fall in when they copy code snippets online. Also what Jack said. We should at least be careful to not feature it prominently in tons of examples.

For something like a logging exporter, it would still be nice to at least take serialization and IO off the request thread instead of blocking?

Reply:
The simple span processor is the only other processor currently defined in the specification, for the sake of completeness it should be supported, but I agree that maybe it's just not featured in the "recommended" example. WDYT?